index.js

```diff
@@ -84,6 +85,27 @@ client.on("message", message => {
 		return message.channel.send(reply);
 	}
 
+	if (!cooldowns.has(command.name)) {
+		cooldowns.set(command.name, new Discord.Collection());
+	}
+
+	const now = Date.now();
+	const timestamps = cooldowns.get(command.name);
+	const cooldownAmount = (command.cooldown || 5) * 1000;
+
+	// Check for cooldown
+	if (timestamps.has(message.author.id)) {
+		const expirationTime = timestamps.get(message.author.id) + cooldownAmount;
+
+		if (now < expirationTime) {
+			const timeLeft = (expirationTime - now) / 1000;
+			return message.reply(`please wait ${timeLeft.toFixed(1)} more second(s) before reusing the \`${command.name}\` command.`);
```

Comment:
nwm czy chcemy tą linijke
bo w sumie zmiejszyło by to spam

Reply:
Co powiesz, żeby wysłać tą wiadomość na PM?

Reply:
podoba mi sie ten pomysl bd nad tym pracowal